README.md

```diff
@@ -92,10 +92,16 @@ If you enter a invalid Snyk token, a pop-up message alerts you that your Snyk to
 If you need a new Snyk token, you can generate a new token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9).
 
 **Red Hat Dependency Analytics Report File Path** :
-
-Specify the local path to create the Red Hat Dependency Analytics report file.
+<br >Specify the local path to create the Red Hat Dependency Analytics report file.
 The default path is `/tmp/redhatDependencyAnalyticsReport.html`.
 
+<br >**IMPORTANT:** 
+<br >The `redHatDependencyAnalyticsReportFilePath` setting name has changed to `reportFilePath`.
+If you had a custom file path set for `redHatDependencyAnalyticsReportFilePath`, then you need to add your custom file path to the `reportFilePath` setting.
```

Comment:
Maybe this should be in changelog and not in README? im thinking that the README should only reflect the current state.

Reply:
It is mentioned in the change log, see line 3.  I'll remove this sentence.